themes/openy_themes/openy_rose/scripts/openy_rose.js

```diff
@@ -261,4 +262,18 @@
     }
   };
 
+  /**
+   * Fill ckeditor table cell padding with value for cellpadding.
+   */
+  Drupal.behaviors.ckeditorTablePadding = {
```

Comment:
What do you think if we move it into one module's library?
This, of course, would apply this behavior to Carnation as well, but it won't break it.

Reply:
OK. Moved code to openy_ckeditor module